### ISSUE TITLE
[PHP] mysql_pconnect - add mysql_connect

### ISSUE THREAD

**Issue by TomasVotruba** [rectorphp/rector#1069]
- [x] http://php.net/manual/en/function.mysql-connect.php

```diff
-mysql_connect()
+mysqli_connect()
```

- [x] http://php.net/manual/en/function.mysql-pconnect.php

```bash
mysqli_connect() with p: host prefix
```

**Comment by TomasVotruba** [comment]
```diff
 $this->_activedb = mysqli_connect('localhost', 'user', 'root');
-mysql_select_db('abc');
+mysqli_select_db($this->_activedb, 'abc');
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_select_db(DB_NAME, $db)
+mysqli_select_db($db, DB_NAME)
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_set_charset('utf8', $db);
+mysqli_set_charset($db, 'utf8');
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_query("QUERY", $db);
+mysqli_query($db, "QUERY");
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_fetch_row
+mysqli_fetch_row
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_fetch_array
+mysqli_fetch_array
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_num_rows
+mysqli_num_rows
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_real_escape_string($code, $link);
+mysqli_real_escape_string($link, $code);
```

**Comment by TomasVotruba** [comment]
```diff
-mysql_query($query, $db);
+mysql_query($db, $query);
```

**Comment by TomasVotruba** [comment]
```diff
-MYSQL_ASSOC
+MYSQLI_ASSOC
```

**Comment by TomasVotruba** [comment]
Resolved by #1072
